aspnetcore/blazor/components/virtualization.md

```diff
@@ -70,8 +70,7 @@ The <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601> com
 
 * Calculates the number of items to render based on the height of the container and the size of the rendered items.
 * Recalculates and rerenders the items as the user scrolls.
-* Only fetches the slice of records from an external API that correspond to the current visible region, instead of downloading all of the data from the collection.
-* Receives a generic <xref:System.Collections.Generic.ICollection%601> for <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.Items?displayProperty=nameWithType>. If a non-generic collection supplies the items (for example, a collection of <xref:System.Data.DataRow>), follow the guidance in the [Item provider delegate](#item-provider-delegate) section to supply the items.
+* Only fetches the slice of records from an external API that correspond to the currently visible region (+ overscan) when `ItemsProvider` is used instead of `Items` (see [Item provider delegate](#item-provider-delegate) section).
```

Comment:
Can we go with "including overscan" in hopes that the localization team understands it better in case the plus sign isn't clear to them. Many of our docs are translated by human workers by hand.

```suggestion
* Only fetches the slice of records from an external API that correspond to the currently visible region, including overscan, when `ItemsProvider` is used instead of `Items` (see [Item provider delegate](#item-provider-delegate) section).
```